src/core/config.py

```diff
@@ -1,22 +1,30 @@
+import os
+from pathlib import Path
 from datetime import datetime
-
 from dotenv import dotenv_values
 
-# ----------------------------------------------------------------------------------------------------- #
-# Getting variables from .env                                                                           #
-# ----------------------------------------------------------------------------------------------------- #
+BASE_DIR = Path(__file__).parent.parent.parent
```

Comment:
```suggestion
BASE_DIR = Path(__file__).resolve().parent.parent
```
логи в src оставим. так как когда скопируем в докер, возможно прав у юзера выше src не будет на запись и задание файлов. также resolve - нормалиовать и получить полный пас